includes/modules/pages/featured_categories/header_php.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Featured Categories
+ *
+ * @copyright Copyright 2003-2024 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: DrByte 2024 Jan 31 Modified in v2.0.0-beta1 $
+ * based on Featured Products
+ */
+
+// This should be first line of the script:
+$zco_notifier->notify('NOTIFY_HEADER_START_FEATURED_CATEGORIES');
+
+require DIR_WS_MODULES . zen_get_module_directory('require_languages.php');
+
+// load extra language strings used by category_listing module
+$languageLoader->setCurrentPage('index');
+$languageLoader->loadLanguageForView();
+
+$breadcrumb->add(NAVBAR_TITLE);
+
+$listing_sql = "SELECT p.categories_id, p.categories_image, pd.categories_name
+                FROM " . TABLE_CATEGORIES . " p
+                LEFT JOIN " . TABLE_FEATURED_CATEGORIES . " f ON p.categories_id = f.categories_id
+                LEFT JOIN " . TABLE_CATEGORIES_DESCRIPTION . " pd ON p.categories_id = pd.categories_id
+                AND pd.language_id = " . (int)$_SESSION['languages_id'] . "
```

Comment:
Use `c` and `cd` as the aliases instead of `p` and `pd`; those are references to category information, not product.